build.gradle.kts

```diff
@@ -89,3 +84,50 @@ subprojects {
     }
   }
 }
+
+allprojects {
+  tasks.withType<DokkaTask>().configureEach {
+    dokkaSourceSets.configureEach {
+      reportUndocumented.set(false)
+      skipDeprecated.set(true)
+      jdkVersion.set(8)
+      perPackageOption {
+        matchingRegex.set("app\\.cash\\.zipline\\.internal\\.*")
+        suppress.set(true)
+      }
+    }
+    if (name == "dokkaGfm") {
+      outputDirectory.set(project.file("${project.rootDir}/docs/0.x"))
+    }
+  }
+
+  plugins.withId("com.vanniktech.maven.publish.base") {
+    configure<MavenPublishBaseExtension> {
+      publishToMavenCentral(SonatypeHost.DEFAULT)
+      signAllPublications()
+      pom {
+        description.set("Runs Kotlin/JS libraries in Kotlin/JVM and Kotlin/Native programs")
+        name.set(project.name)
+        url.set("https://github.com/cashapp/zipline/")
+        licenses {
+          license {
+            name.set("The Apache Software License, Version 2.0")
+            url.set("http://www.apache.org/licenses/LICENSE-2.0.txt")
+            distribution.set("repo")
+          }
+        }
+        developers {
+          developer {
+            id.set("square")
```

Comment:
```suggestion
            id.set("cashapp")
```

Reply:
I hope we’re not doing this .. . . so far as I know there’s no corporate entity for `cashapp` and we shouldn’t assign ownership to it?

Reply:
We do this on almost every cashapp project. There's no legal requirement here or anything. I think it ties it more closely to the GitHub org than anything.

Reply:
Works for me.